app/tests/test_routes.py

```diff
@@ -10,6 +10,238 @@ def test_base_route():
     url = '/'
 
     response = client.get(url)
-
     assert response.status_code == 200
     assert response.get_data() == b'try the predict route it is great!'
+
+def test_predict_allFields():
+    app = Flask(__name__)
+    configure_routes(app)
+    client = app.test_client()
+    data = {
+        "sex": "F",
+        "age" : 18,
+        "address": "U",
+        "famsize": "GT3",
+        "Pstatus": "A",
+        "Medu": 4,
+        "Fedu": 4,
+        "Mjob:": "at_home",
+        "Fjob": "teacher",
+        "reason" : "course",
+        "guardian": "mother",
+        "traveltime": 2,
+        "studytime": 2,
+        "failures": 0,
+        "schoolsup": True,
+        "famsup": False,
+        "paid": False,
+        "activities": False,
+        "nursery": True,
+        "higher": True,
+        "internet": False,
+        "romantic": False,
+        "famrel": 4,
+        "freetime": 3,
+        "goout": 4,
+        "Dalc": 1,
+        "Wcal": 1,
+        "health": 3,
+        "absences": 6
+    }
+    response = client.get('/predict', json = data)
+    output = response.get_data()
+    assert response.status_code == 200
+    assert output == data
```

Comment:
The output is not the same as the input here. Could you please update the test so that it verifies the output evaluates to either `{"isQualified": true}` or `{"isQualified": false}`?